repl/src/test/scala/org/apache/spark/repl/ExecutorClassLoaderSuite.scala

```diff
@@ -99,6 +107,32 @@ class ExecutorClassLoaderSuite
     }
   }
 
+  test("resource from parent") {
+    val parentLoader = new URLClassLoader(urls2, null)
+    val classLoader = new ExecutorClassLoader(new SparkConf(), url1, parentLoader, true)
+    val resourceName: String = "fake-resource.txt"
+    Option(classLoader.getResource(resourceName)) match {
```

Comment:
I'd simplify this a bit.

    val is = classLoader.getResourceAsStream(resourceName)
    assert(is != null)
    val content = Source.fromInputStream(is, "UTF-8").getLines().first
    assert(content.contains(...))